tests/config/base_ospdo/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+- openstack_control_plane_ospdo.yaml
+patchesStrategicMerge:
+- tls_overrides.yaml
```

Comment:
As mentioned above, we don't use kustomization anymore, and the tls_overrides.yaml file does not exist. See #592 -- there is now openstack_control_plane.j2. So the openstack_control_plane_ospdo.yaml file should be openstack_control_plane_ospdo.j2 (or openstack_control_plane_ospdo.yaml.j2) and it should include the tls section like this:

https://github.com/openstack-k8s-operators/data-plane-adoption/pull/592/files#diff-a36bfb0872744e4139bdd9164e4e34c8cdd38b6e24cb9c01690101421063b0cb

Reply:
ok , patching